pages/devops-checklist/_sub-hero.html

```diff
@@ -0,0 +1,15 @@
+<div class="row row-page-header">
+  <div class="col-xs-12">
+    <h2>Everything you need to do before you go live</h2>
+    <p>
+      This checklist is your guide to the best practices for deploying secure, scalable, and highly available
+      infrastructure in AWS. Before you go live, go through each item, and make sure you haven't missed anything
+      important!
+    </p>
+    <ol>
+      {% for category in site.data.devops-checklist %}
+        <li><a href="#{{ category.category | slugify }}">{{ category.category}}</a></li>
+      {% endfor %}
+    </ol>
```

Comment:
Consider adding a little note informing users that checked boxes will persist in local storage across browser sessions.

Reply:
Done: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/45/commits/fed86ece7bbdf16a1fe88220450df7eec8a94c60. I also added "check all", "uncheck all", etc controls in that commit.